sdk/Lusid.Sdk.Tests/tutorials/Instruments/CreditDefaultSwapCashFlow.cs

```diff
@@ -0,0 +1,134 @@
+        };
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            _apiFactory = TestLusidApiFactoryBuilder.CreateApiFactory("secrets.json");
+            _transactionPortfoliosApi = _apiFactory.Api<ITransactionPortfoliosApi>();
+            _portfoliosApi = _apiFactory.Api<IPortfoliosApi>();
+            _recipeApi = _apiFactory.Api<IConfigurationRecipeApi>();
+            _aggregationApi = _apiFactory.Api<IAggregationApi>();
+            _testDataUtilities = new TestDataUtilities(
+                _apiFactory.Api<ITransactionPortfoliosApi>(),
+                _apiFactory.Api<IInstrumentsApi>(),
+                _apiFactory.Api<IQuotesApi>(),
+                _apiFactory.Api<IComplexMarketDataApi>());
+        }
+        
+        [Test]
+        public void ExamplePortfolioCashFlowsForCreditDefaultSwaps()
+        {
+            // CREATE portfolio
+            var portfolioScope = Guid.NewGuid().ToString();
+            var portfolioId = _testDataUtilities.CreateTransactionPortfolio(portfolioScope);
+
+            // CREATE CDS
+            var cds = InstrumentExamples.CreateExampleCreditDefaultSwap() as CreditDefaultSwap;
+        
+            // UPSERT CDS to portfolio and populating stores with required market data
+            var effectiveAt = new DateTimeOffset(2020, 2, 23, 0, 0, 0, TimeSpan.Zero);
+            _testDataUtilities.AddInstrumentsTransactionPortfolioAndPopulateRequiredMarketData(
+                portfolioScope, 
+                portfolioId,
+                effectiveAt,
+                effectiveAt,
+                cds);
+            
+            // UPSERT CDS spread curve before upserting recipe
+            _testDataUtilities.UpsertCdsSpreadCurves(portfolioScope, effectiveAt, cds.Ticker, cds.FlowConventions.Currency, cds.ProtectionDetailSpecification.Seniority, 
+                cds.ProtectionDetailSpecification.RestructuringType);
+            
+            // CREATE and upsert recipe specifying discount pricing model
+            var discountingRecipeCode = "DiscountingRecipe";
+            CreateAndUpsertRecipe(discountingRecipeCode, portfolioScope, ModelSelection.ModelEnum.Discounting);
+            
+            // CALL api to get cashflows at maturity
+            var maturity = cds.MaturityDate;
+            var cashFlowsAtMaturity = _transactionPortfoliosApi.GetPortfolioCashFlows(
+                portfolioScope,
+                portfolioId,
+                maturity.AddMilliseconds(-1),
+                maturity.AddMilliseconds(-1),
+                maturity.AddMilliseconds(1),
+                null,
+                null,
+                portfolioScope,
+                discountingRecipeCode);
+            
+            // CHECK correct number of cashflow at CDS maturity: There is 1 cash flows corresponding to the last coupon amount.
+            var expectedNumber = 1;
+            Assert.That(cashFlowsAtMaturity.Values.Count, Is.EqualTo(expectedNumber));
```

Comment:
Can we add something to guide users what cash flows we are returning here? The example looks good, but it's not clear what this 1 cash flow represents. Perhaps even a bit more description on the comment there could help.

Reply:
Added further comment for clarification.